mmedit/datasets/pipelines/re_sampling.py

```diff
@@ -7,6 +7,73 @@
 from ..registry import PIPELINES
 
 
+@PIPELINES.register_module()
+class ReSampling:
+    """Resize image.
```

Comment:
What's the difference between resample and resize

Reply:
No difference here.